src/Processors/Transforms/Streaming/AggregatingHelper.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include <vector>
+
+namespace DB
+{
+class Block;
+class Chunk;
+
+namespace Streaming
+{
+struct AggregatingTransformParams;
+struct AggregatedDataVariants;
+using AggregatedDataVariantsPtr = std::shared_ptr<AggregatedDataVariants>;
+using ManyAggregatedDataVariants = std::vector<AggregatedDataVariantsPtr>;
+
+using RetractedDataVariants = AggregatedDataVariants;
+using RetractedDataVariantsPtr = std::shared_ptr<RetractedDataVariants>;
+using ManyRetractedDataVariants = ManyAggregatedDataVariants;
+
+using ChunkPair = std::pair<Chunk, Chunk>;
+
+struct AggregatingHelper
```

Comment:
Why use struct？ it does not contain any member variables.

Reply:
In AggregatingHelper, almost methods invoke private methods of Aggregator, so here use a friend struct instead of a lot of friend functions